examples/browser/webpack.config.js

```diff
@@ -1,4 +1,4 @@
-/**
+AAAA/**
```

Comment:
Please revert.

Reply:
Sorry about that, removed in the next commits now